packages/neuron-ui/src/components/ApproveMultisigTxDialog/approveMultisigTx.module.scss

```diff
@@ -0,0 +1,180 @@
+@import '../../styles/mixin.scss';
+
+.dialogContainer {
+  padding: 0;
+}
+
+.container {
+  width: 680px;
+
+  .header {
+    line-height: 48px;
+    padding: 0 16px;
+    font-size: 14px;
+    display: flex;
+    align-items: center;
+    background: var(--table-head-background-color);
+    color: var(--secondary-text-color);
+    border-bottom: 1px solid var(--table-head-border-color);
+
+    .tip {
+      left: calc(100% - 240px) !important;
```

Comment:
It is necessary

Reply:
> It is necessary

As long as the existing value of left is not `!important` or an inline style, it is unnecessary.
The reason why the style did not take effect is simply because the selector's weight is not high enough.
There are various ways to change the weight, such as in this case, the original built-in selector is `.tipWithNode[data-has-trigger=true][data-placement=bottom]:not([data-trigger-next-to-child=true]) .tip`, which has a weight of `5 * 10 = 50`, while the current selector only has a weight of 30. Here are some examples of how to increase weight:

1. `.tip.tip.tip` - adding multiple class selectors

2. `.tip:not([a][b])` - adding invalid attribute selectors

Alternatively, you can decrease the weight of the previous selector:

Move `.tip` from `.tipWithNode` to the top level in `Tooltip/tooltip.module.scss`, so the weight will only be 10.

I would recommend using the second solution, as common components should be designed to make their styles easy to override.

I hope to never use `!important`, as it can break the maintainability of CSS.